test/pathforaSpec.js

```diff
@@ -3899,6 +3899,53 @@ describe('Widgets', function () {
     });
   });
 
+  describe('when metaContains is set', function () {
+    var id = 'meta-contains-test';
+    var modal;
+
+    beforeEach(function () {
+      modal = new pathfora.Message({
+        layout: 'modal',
+        id: id,
+        headline: 'This will show...',
+        msg: '... if your meta tags match the conditions',
+        displayConditions: {
+          metaContains: [
+            {
+              property: 'og:type',
+              content: 'product'
+            },
+            {
+              property: 'og:locale',
+              content: 'en_US'
+            }
+          ]
+        }
+      });
+    });
+
+    it('should not show if the meta rules are not met', function () {
+      pathfora.initializeWidgets([modal]);
+      expect($('#' + id).length).toBe(0);
+    });
+
+    it('should not show if a single meta rule is met', function () {
+      $('head').append('<meta property="og:type" content="product">');
+      $('head').append('<meta name="description" content="cool description here">');
+      pathfora.initializeWidgets([modal]);
+      expect($('#' + id).length).toBe(1);
```

Comment:
in the case of this one dont think its just a typo. the module will show if 1 of 2 conditions are met...which is why this test is passing :) so might want to look at that piece. just verified on a local test also.